.gitignore

```diff
@@ -12,6 +12,8 @@ hip_build/
 OUTPUT_IMAGES*
 OUTPUT_PERFORMANCE*
 QA_RESULTS*
+OUTPUT_AUDIO*
```

Comment:
Why output audio is added?
we are not generating any audio outputs right?

Reply:
When we discussed we plan to log the PASS OR FAIL IN an text file right
I am adding that txt file in OUTPUT_AUDIO* folder

Reply:
@sampath1117  can you please confirm this

Reply:
@r-abishek For image unit tests we are having output folder as OUTPUT_IMAGE_HOST , so for the audio I am using OUTPUT_AUDIO_HOST 
is it Okay or we need some different folder name?

Reply:
Similar folder name is fine yes, but only if its necessary to store actual outputs in it.
I was thinking PASS/FAIL is part of QA testing logged inside QA_results.txt inside a folder that starts with QA_RESULTS_* and not OUTPUT_*
Pls clarify